src/test/java/org/folio/search/controller/SearchInstanceIT.java

```diff
@@ -249,6 +250,8 @@ private static Stream<Arguments> testDataProvider() {
       arguments("contributors.name = {value}", "Anton*"),
       arguments("contributors.name = {value}", "*rmelen, Frank"),
 
+      arguments("contributors.authorityId == {value}", "55294032-fcf6-45cc-b6da-4420a61ef72c"),
```

Comment:
Please add a test for the alias
`authorityId == {value}`

Reply:
done